sdk/lib/crt/stdlib/rand_s.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     rand_s implementation
+ * COPYRIGHT:   Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#include <precomp.h>
+#include <ksecclient.h>
+
+/*!
```

Comment:
```suggestion
/*
```
Assuming that the exclamation mark has no meaning, but was a typo.

Reply:
If you mean 'doxygen' with 'no meaning' then yes.